source/features/forked-to.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const getCacheKey = (repo: string): string => `forked-to:${repo}`;
+
+async function showForks(): Promise<void> {
+	const cached = await getValidatedCache(getSourceRepo());
+	const pageHeader = select('.pagehead h1.public')!;
+	for (const fork of cached.filter(fork => fork !== getRepoURL())) {
+		pageHeader.append(
+			<span className="fork-flag rgh-forked">forked to&nbsp;
+				<a href={`/${fork}`}>{fork}</a>
+			</span>
+		);
+	}
+}
+
+function rememberCurrentFork(): void {
+	if (!isOwnRepo()) {
+		return;
+	}
+
+	const forkedRepo = findForkedRepo();
+	if (forkedRepo) {
+		addAndStoreCache(forkedRepo, getRepoURL());
+	}
+}
+
+function watchForkDialog(): void {
+	const forkDialog = select<HTMLElement>('details-dialog[src*="/fork"]')!;
```

Comment:
All the feedback I sent before applies to other parts of the file as well. I can’t review every single line. This PR has been sloppy. Please clean it up and open a new PR

Reply:
Webpack compiles, lint succeeds, the PR does exactly what it should do, in the way you specified it should be done, without code style guide. I try to follow other features syntax, but you have a very opinionated way of writing code. I'm not saying that that is bad, and the feedback is great.

See https://github.com/sindresorhus/refined-github/pull/2153

Reply:
I haven't followed the review here, but what @bfred-it means is that when he reviews some change, and you fix it, you should look for other places where the same fix or similar fix could be applied. You should also look if there's any way to improve/simplify the code after the changes.